History/js/new.js

```diff
@@ -195,7 +197,7 @@ var callbacks = {
       
       tableBody.appendChild(tr);
       
-    }
+    } //for loop completion.
```

Comment:
Sorry for forgetting to delete that comment. I got confused at a certain point as to what that bracket was for. I'll remove that.